client/api_object.go

```diff
@@ -1031,59 +1037,6 @@ func (c *Client) ListObjects(ctx context.Context, bucketName string, opts types.
 	return listObjectsResult, nil
 }
 
-// GetCreateObjectApproval returns the signature info for the approval of preCreating resources
```

Comment:
// Deprecated: GetCreateObjectApproval returns the signature info for the approval of preCreating resources

No need to delete this API

Reply:
fixed

Reply:
I mean change the comment to other than delete this line:

// Deprecated: GetCreateObjectApproval returns the signature info for the approval of preCreating resources

Reply:
And the same to GetCreateBucketApproval API

Reply:
fixed